ci/Jenkinsfile.combined

```diff
@@ -25,9 +25,11 @@ pipeline {
     stage('Build') {
       parallel {
         stage('MacOS') { steps { script {
+          when { expression { cmn.getBuildType() != 'release' } } 
```

Comment:
The `when` block should be on the same level as the `steps` block, see example:
https://jenkins.io/doc/book/pipeline/syntax/#when-example